PokemonRampUp/Model/Move.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  Move.swift
+//  PokemonRampUp
+//
+//  Created by Thushen Mohanarajah on 2023-02-14.
+//
+
+import Foundation
+
+// MARK: - Move
+struct Move: Decodable {
+    let move: Species
```

Comment:
What is move called Species?

Reply:
```
struct Species: Decodable {
    let name: String
    let url: String
    
    init(name: String,
         url: String
    ){
        self.name = name
        self.url = url
    }
}
```

`Species` is basically a generic class that is used in multiple cases. For example, take a look at this JSON blocks send from PokeAPI:

In `moves` object in JSON file
```
    "moves": [
        {
            "move": {
                "name": "mega-punch",
                "url": "https://pokeapi.co/api/v2/move/5/"
            },
...
```

In `Types` object in JSON file
```
    "types": [
        {
            "slot": 1,
            "type": {
                "name": "electric",
                "url": "https://pokeapi.co/api/v2/type/13/"
            }
        }
    ],
...

```

Do you see how that there is inner object that contains a `name` and `url` variable. That is why I have a model called `Species`. I am going to change the name of the file to `NameAndURL` . I think it is a better name but, feel free to share your thoughts on a name.